src/modules/job-manager/jobtap.c

```diff
@@ -333,6 +377,7 @@ int jobtap_load (struct jobtap *jobtap,
                   "Plugin did not set name in flux_plugin_init");
         goto error;
     }
+done:
     jobtap->plugin = p;
     jobtap->plugin_path = strdup (path);
```

Comment:
check for `strdup()` failre

Reply:
I'll just remove jobtap->plugin_path, it is unused for now.